<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContactListMainScreen</class>
 <widget class="QMainWindow" name="ContactListMainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>715</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="Backgrondframe">
      <property name="styleSheet">
       <string notr="true">QFrame{
	
	background-color: qlineargradient(spread:pad, x1:0.539773, y1:1, x2:0.528636, y2:0.266, stop:0 rgba(158, 139, 101, 255), stop:1 rgba(57, 66, 136, 255));
	color: rgb(230, 230, 230);
	border-radius: 12px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_mainTitle">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>981</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>33</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 238);
background: transparent;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;EMPLOYEE&lt;/span&gt; | CONTACT LIST&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_firstName">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>130</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 3px solid rgb(0, 108, 154);
	border-radius: 12px;
	color: rgb(0, 0, 0);
	padding-left: 20px;
	padding-right: 20px; 
	background-color: rgb(255, 255, 255);
}

QLineEdit:hover {
	border: 2px solid rgb(0, 165, 230);
}

QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	
	background-color: rgb(198, 198, 198);
}</string>
       </property>
       <property name="placeholderText">
        <string>Enter Your First Name Here</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_firstName">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 238);
background: transparent;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;First Name:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_emailAddress">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 238);
background: transparent;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Email Address:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_emailAddress">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>190</y>
         <width>751</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 3px solid rgb(0, 108, 154);
	border-radius: 12px;
	color: rgb(0, 0, 0);
	padding-left: 20px;
	padding-right: 20px; 
	background-color: rgb(255, 255, 255);
}

QLineEdit:hover {
	border: 2px solid rgb(0, 165, 230);
}

QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	
	background-color: rgb(198, 198, 198);
}</string>
       </property>
       <property name="placeholderText">
        <string>Input Your Email Address Here</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_lastName">
       <property name="geometry">
        <rect>
         <x>650</x>
         <y>130</y>
         <width>291</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 3px solid rgb(0, 108, 154);
	border-radius: 12px;
	color: rgb(0, 0, 0);
	padding-left: 20px;
	padding-right: 20px; 
	background-color: rgb(255, 255, 255);
}

QLineEdit:hover {
	border: 2px solid rgb(0, 165, 230);
}

QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	
	background-color: rgb(198, 198, 198);
}</string>
       </property>
       <property name="placeholderText">
        <string>Enter Your Last Name Here</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_lastName">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>110</y>
         <width>141</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 238);
background: transparent;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Last Name:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_phoneNumber">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>171</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 238);
background: transparent;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Phone Number:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_jobPosition">
       <property name="geometry">
        <rect>
         <x>690</x>
         <y>250</y>
         <width>241</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Select Job Title Here</string>
       </property>
       <item>
        <property name="text">
         <string>Receptionist</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Web Developer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Data Analyst</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Software Engineer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Designer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Marketing Manager</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Illustrator</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Network Administrator</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Graphic Designer</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Brand Ambassador</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_phoneNumber1">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>250</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	setMaxLength: 3;
	border: 3px solid rgb(0, 108, 154);
	border-radius: 12px;
	color: rgb(0, 0, 0);
	padding-left: 20px;
	padding-right: 20px; 
	background-color: rgb(255, 255, 255);
}

QLineEdit:hover {
	border: 2px solid rgb(0, 165, 230);
}

QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	
	background-color: rgb(198, 198, 198);
}

QLineEdit::setMaxLength(3);
QLineEdit::setMaxLenght (3);</string>
       </property>
       <property name="placeholderText">
        <string>###</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_phoneNumber2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>250</y>
         <width>91</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 3px solid rgb(0, 108, 154);
	border-radius: 12px;
	color: rgb(0, 0, 0);
	padding-left: 20px;
	padding-right: 20px; 
	background-color: rgb(255, 255, 255);
}

QLineEdit:hover {
	border: 2px solid rgb(0, 165, 230);
}

QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	
	background-color: rgb(198, 198, 198);
}</string>
       </property>
       <property name="placeholderText">
        <string>###</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_phoneNumber3">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>250</y>
         <width>101</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 3px solid rgb(0, 108, 154);
	border-radius: 12px;
	color: rgb(0, 0, 0);
	padding-left: 20px;
	padding-right: 20px; 
	background-color: rgb(255, 255, 255);
}

QLineEdit:hover {
	border: 2px solid rgb(0, 165, 230);
}

QLineEdit:focus{
	border: 2px solid rgb(85, 170, 255);
	
	background-color: rgb(198, 198, 198);
}</string>
       </property>
       <property name="placeholderText">
        <string>####</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_hyphen1">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>240</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 238);
background: transparent;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_hyphen2">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>240</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 238);
background: transparent;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;-&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_jobPosition">
       <property name="geometry">
        <rect>
         <x>550</x>
         <y>220</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 238);
background: transparent;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Job Position:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_save">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 3px solid rgb(0, 108, 154);
	border-radius: 10px;
	
	color: rgb(255, 175, 46);
	padding-left: 5px;
	padding-right: 5px; 
	background-color: rgb(55, 55, 55);
}
</string>
       </property>
       <property name="text">
        <string>SAVE</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_information">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>300</y>
         <width>921</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 4);
background: transparent;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QTableWidget" name="tableWidget_display">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>370</y>
         <width>841</width>
         <height>241</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 3px solid rgb(0, 108, 154);
border-radius: 10px;
background-color: qlineargradient(spread:pad, x1:0.528, y1:0, x2:0.465909, y2:1, stop:0 rgba(92, 179, 255, 255), stop:0.982955 rgba(218, 218, 218, 255), stop:1 rgba(225, 225, 225, 255));
color: rgb(0, 0, 0);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_clear">
       <property name="geometry">
        <rect>
         <x>840</x>
         <y>40</y>
         <width>91</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>15</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	border: 3px solid rgb(0, 108, 154);
	border-radius: 10px;
	
	color: rgb(255, 175, 46);
	padding-left: 5px;
	padding-right: 5px; 
	background-color: rgb(55, 55, 55);
}
</string>
       </property>
       <property name="text">
        <string>CLEAR</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
